### ISSUE TITLE
code sync crashes when /tmp is on a different partion(?)

### ISSUE THREAD

**Issue by rwos** [convox/rack#1469]
I have `/tmp` on a ram disk and the code sync doesn't like that:

```
$ mount | fgrep /tmp
tmpfs on /tmp type tmpfs (rw,noatime)
$ uname -a
Linux easybox 4.3.0-1-amd64 #1 SMP Debian 4.3.3-7 (2016-01-19) x86_64 GNU/Linux
$ convox -v
client: 20161208160629
```

During `convox start` there's one log entry that looks like this:
```
convox       │ error: rename /tmp/foo.bar841595221 /home/richard/project/foo.bar: invalid cross-device link
```

and obviously nothing syncs

**Comment by ddollar** [comment]
Looks like the culprit is here: https://github.com/convox/rack/blob/master/sync/sync.go#L220

I'm curious if you are able to `mv` files between your `/tmp` and other filesystems?

**Comment by rwos** [comment]
@ddollar `mv` works, yes.

**Comment by mattmanning** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [code sync crashes when /tmp is on a different partion(?)](https://trello.com/c/FDuqBXVI/1365-code-sync-crashes-when-tmp-is-on-a-different-partion)

**Comment by MiguelMoll** [comment]
Addressed via #1472. CLI version `20161215133928`.
